tests/main/interfaces-timeserver-control/task.yaml

```diff
@@ -1,5 +1,10 @@
 summary: Check that timeserver interface works
 
+# This makes only sense on ubuntu-core because this snap will run inside
+# the core environment on a classic system we cannot really use this
+# on classic
```

Comment:
This comment is unclear to me ('this' used too many times). Are you saying the interface isn't useful on classic?

Reply:
Sorry for `this` ;) I will investigate and clarify the comment further.

Reply:
I think I was actually wrong here, it looks like the breaking is happening because of the missing AssumedApparmorLabel